whole-repo-tests/test_mypy.py

```diff
@@ -31,15 +31,15 @@ def test_mypy_passes_on_hypothesis_strict():
 
 
 def get_mypy_output(fname, *extra_args):
-    return subprocess.Popen(
+    return subprocess.run(
         [tool_path("mypy"), *extra_args, fname],
-        stdout=subprocess.PIPE,
         encoding="utf-8",
-        universal_newlines=True,
+        capture_output=True,
+        text=True,
         # We set the MYPYPATH explicitly, because PEP561 discovery wasn't
         # working in CI as of mypy==0.730 - hopefully a temporary workaround.
         env=dict(os.environ, MYPYPATH=PYTHON_SRC),
```

Comment:
I should try removing this, and maybe grep the whole repo for "workaround" 🤔

Reply:
One thing I did early on in qutebrowser is to mark all workarounds with `WORKAROUND:`, ideally with version numbers and/or links to bug reports. Makes it a joy to grep for those and drop them all when updating minimum versions of dependencies!